homeassistant/components/kraken/const.py

```diff
@@ -8,21 +20,21 @@
 
 DOMAIN = "kraken"
 
-SENSOR_TYPES = [
-    {"name": "ask", "enabled_by_default": True},
-    {"name": "ask_volume", "enabled_by_default": False},
-    {"name": "bid", "enabled_by_default": True},
-    {"name": "bid_volume", "enabled_by_default": False},
-    {"name": "volume_today", "enabled_by_default": False},
-    {"name": "volume_last_24h", "enabled_by_default": False},
-    {"name": "volume_weighted_average_today", "enabled_by_default": False},
-    {"name": "volume_weighted_average_last_24h", "enabled_by_default": False},
-    {"name": "number_of_trades_today", "enabled_by_default": False},
-    {"name": "number_of_trades_last_24h", "enabled_by_default": False},
-    {"name": "last_trade_closed", "enabled_by_default": False},
-    {"name": "low_today", "enabled_by_default": True},
-    {"name": "low_last_24h", "enabled_by_default": False},
-    {"name": "high_today", "enabled_by_default": True},
-    {"name": "high_last_24h", "enabled_by_default": False},
-    {"name": "opening_price_today", "enabled_by_default": False},
+SENSOR_TYPES: List[SensorType] = [
+    SensorType(name="ask", enabled_by_default=True),
```

Comment:
Using `TypedDict` constructor was not necessary but it's ok either way. I prefer creating it as a regular dict.

Reply:
I don't think we should use the TypedDict to actually create the exit for production. We don't do that anywhere.

Reply:
So completely remove the typed dict and revert back to how it was before?

Reply:
We use TypedDict to type dicts that have a specific set of string keys with different value types. The actual dict is created as normal.

Reply:
I reverted back to the original creation